cluster/cluster.go

```diff
@@ -0,0 +1,83 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/hashicorp/raft"
+	"github.com/narvikd/errorskit"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"log"
+	"nubedb/api/proto"
+	"nubedb/cluster/consensus/fsm"
+	"nubedb/internal/config"
+	"time"
+)
+
+func Execute(cfg config.Config, consensus *raft.Raft, payload *fsm.Payload) error {
+	payloadData, errMarshal := json.Marshal(&payload)
```

Comment:
There's double marshalling, here and in the proto service layer.
For small data is not a problem, but what if it's a large chunk?

Reply:
Fixed: https://github.com/narvikd/nubedb/pull/10/commits/90decfd1179e510ab4238d4e88b6f349450ebe91